USAGE.md

```diff
@@ -1,4 +1,4 @@
-# Casper Event Sidecar USAGE
+# Casper Sidecar USAGE
```

Comment:
This is a generic comment to all markdowns, since I was not able to add a comment to a line that was not changed in this PR.

However, for example, [here](https://github.com/casper-network/casper-sidecar/pull/303/files#diff-58a825c70bccfbe691d7ddbddb133275f4f657de6aeb4a502fa9fce279f8bf8dR185) we mark the `bash` code with `json`.

It'll be good to just quickly review all the quoted code and see if we have correct language specifiers applied.

Reply:
Thanks for catching this. I've cleaned up all files except LEGACY_SSE_EMULATION.md, which will be addressed with another PR.